### PR TITLE
Blazeface

### PR DESCRIPTION
**Pull request by yuehaixiao**
add facebox detector function.
add density_prior_box in blazeface.
add 5x5 kernel option in blazeface.

### CODE REVIEW

File: PaddleCV/PaddleDetection/ppdet/modeling/architectures/blazeface.py [hunk 1]
```diff
@@ -74,18 +74,26 @@ def build(self, feed_vars, mode='train'):
...
             pred = self.output_decoder(locs, confs, box, box_var)
             return {'bbox': pred}
 
-    def _multi_box_head(self, inputs, image, num_classes=2):
-
+    def _multi_box_head(self,
+                        inputs,
+                        image,
+                        num_classes=2,
+                        use_density_prior_box=True):
```
Comment (still-wait) [inline PaddleCV/PaddleDetection/ppdet/modeling/architectures/blazeface.py]: default use_density_prior_box=False?

File: PaddleCV/PaddleDetection/ppdet/utils/eval_utils.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
 import paddle.fluid as fluid
 
-from ppdet.utils.voc_eval import bbox_eval as voc_bbox_eval
+#from ppdet.utils.voc_eval import bbox_eval as voc_bbox_eval
```
Comment (still-wait) [inline PaddleCV/PaddleDetection/ppdet/utils/eval_utils.py]: 这个不能注释

File: PaddleCV/PaddleDetection/tools/train.py [hunk 3]
```diff
@@ -23,13 +23,14 @@
 import datetime
 from collections import deque
 
+
```
Comment (still-wait) [inline PaddleCV/PaddleDetection/tools/train.py]: train.py可以忽略不必commit

File: PaddleCV/PaddleDetection/ppdet/modeling/architectures/blazeface.py (hunk 1 above)
Comment (yuehaixiao) [reply PaddleCV/PaddleDetection/ppdet/modeling/architectures/blazeface.py]: I set 'False' for model test. 
It's best to set this option to config/*.yml files.

File: PaddleCV/PaddleDetection/.gitignore [hunk 4]
```diff
@@ -56,3 +56,12 @@ coverage.xml
 /docs/_build/
 
 *.json
+
```
Comment (still-wait) [inline PaddleCV/PaddleDetection/.gitignore]: 此文件可忽略不commit

File: PaddleCV/PaddleDetection/configs/blazeface.yml [hunk 5]
```diff
@@ -9,9 +9,9 @@ snapshot_iter: 10000
 log_smooth_window: 20
 log_iter: 20
 metric: WIDERFACE
+save_dir: output_blazelite_128x128_priorbox_5x5_190903
```
Comment (still-wait) [inline PaddleCV/PaddleDetection/configs/blazeface.yml]: 此文件可忽略不commit

File: PaddleCV/PaddleDetection/ppdet/data/data_feed.py [hunk 6]
```diff
@@ -1033,6 +1033,7 @@ def __init__(self,
                  dataset=WiderFaceDataSet().__dict__,
                  fields=['image', 'gt_box', 'gt_label'],
                  image_shape=[3, 128, 128],
+                 #image_shape=[3, 300, 300],
```
Comment (still-wait) [inline PaddleCV/PaddleDetection/ppdet/data/data_feed.py]: 同上

**Pull request merged**